win32ss/gdi/gdi32/objects/bitmap.c

```diff
@@ -666,6 +666,10 @@ SetDIBitsToDevice(
     UINT cjBmpScanSize = 0;
     BOOL Hit = FALSE;
     PVOID pvSafeBits = (PVOID) Bits;
+    UINT bmiHeight;
+
+    #define aligned(Address, Count) \
```

Comment:
It could be of interest to use `ALIGN_DOWN_BY` and co. macros here.

Reply:
I am not familiar with these macros. Can you suggest how they could be used as a direct replacement for this code? Thanks.

Reply:
No response to my question. This works as it is. Resolving now. Thanks.